src/components/VerticalNavigation/VerticalNavigationBrand.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const VerticalNavigationBrand = props => {
+  const { title, href, iconImg, children } = props;
+  // The img prop is just a shorthand for the titleImg prop.
+  // When also using iconImg, it is less confusing to pass titleImg instead of img.
+  const titleImg = props.titleImg || props.img;
+
+  const brandChildren = children || (
+    <span>
+      {iconImg && (
+        <img className="navbar-brand-icon" src={iconImg} alt={title} />
+      )}
+      {titleImg && (
+        <img className="navbar-brand-name" src={titleImg} alt={title} />
+      )}
+      {!titleImg && title && <span className="navbar-brand-txt">{title}</span>}
+    </span>
+  );
+  return href ? (
+    <a href={href} className="navbar-brand">
```

Comment:
would be nice in the future to maybe expose the `onClick` of the `a` too... @jeff-phillips-18 do you have any thoughts? I'm fine either way for now...

Reply:
Yes, leaving nav open when menus are pinnable makes sense to me @mturley . If everyone agrees with this approach, I can file an issue to make the Core behavior consistent.

Reply:
For the brand icon/text, I agree it should be able to use an href, an onClick callback, or nothing at all.

Reply:
Ok I'll make sure it is flexible.